static/dbstructure.config.php

```diff
@@ -159,6 +159,17 @@
 			"email" => ["email(64)"],
 		]
 	],
+	"user-gserver" => [
+		"comment" => "User settings about remote servers",
+		"fields" => [
+			"uid" => ["type" => "mediumint unsigned", "not null" => "1", "default" => "0", "foreign" => ["user" => "uid"], "comment" => "Owner User id"],
+			"gsid" => ["type" => "int unsigned", "not null" => "1", "default" => "0", "foreign" => ["gserver" => "id"], "comment" => "Gserver id"],
+			"muted" => ["type" => "boolean", "not null" => "1", "default" => "0", "comment" => "server accounts are muted for the user"],
```

Comment:
What theme are you referring to?

Reply:
I guess "schema" is the better word.

Reply:
Very well, what schema are you referring to? What do you think I should rename this field to?

Reply:
I mean that we should use the same words that we use for contacts.

Reply:
Got it. None of the words we use for contacts matches this feature. Collapse and Ignore don’t go that far that I know of, and Block goes further. So I thought it warranted a new term to avoid feature confusion.

Reply:
I think that this matches really well the definition of "ignoring", since we are only blocking the display of incoming posts from the given domains.

Reply:
AFAIK we don't show ignored authors on the network page, but only on the display page, don't we?

Reply:
Possibly, which makes it one more place we show ignored authors than muted server authors.

Reply:
So with this PR we have a "mute" feature that mostly acts like the "ignore" feature for contacts and in the future will be called by the Mastodon API endpoints for blocked servers. And since we already have got the functionality in the conversation class to handle blocked, ignored and collapsed contacts, in the future we could easily extent this to the servers as well, so that we would have blocked, ignored, collapsed and muted servers.

I think that this is confusing for the users to understand. That's why I think that for the servers we should stick with the terminology for the contacts.

Reply:
Why not, but then we should align the `ignored` contact behavior to the now-`ignored` server behavior, meaning that posts from an ignored contact should never show in any circumstance, but can still be received as it only affects the display.

Now that I'm writing this, I don't think the check I added in `Protocol\ActivityPub::isAcceptedRequester` is warranted for now-`ignored` domains.